flyteplugins/go/tasks/pluginmachinery/flytek8s/config/config.go

```diff
@@ -203,6 +203,9 @@ type K8sPluginConfig struct {
 
 	// SendObjectEvents indicates whether to send k8s object events in TaskExecutionEvent updates (similar to kubectl get events).
 	SendObjectEvents bool `json:"send-object-events" pflag:",If true, will send k8s object events in TaskExecutionEvent updates."`
+
+	// ImagePullPolicy for the submitted pod.
+	ImagePullPolicy v1.PullPolicy `json:"image-pull-policy" pflag:"-,Image pull policy for all k8s pods created by FlytePropeller."`
```

Comment:
nit: could we move this config to line 159? below to `ImagePullBackoffGracePeriod`